test/start-kubernetes.sh

```diff
@@ -101,45 +100,41 @@ EOF
                         exit 2
                     fi
                 fi
-                unxz ${CLOUD_IMAGE}
+                unxz ${CLOUD_IMAGE} || die "failed to unpack ${CLOUD_IMAGE}"
                 CLOUD_IMAGE=${CLOUD_IMAGE/.xz}
                 ;;
             ubuntu)
                 base_url="https://cloud-images.ubuntu.com/disco/current/"
                 sha_file="SHA256SUMS"
-                curl -s -O ${base_url}/${CLOUD_IMAGE}.xz
-                curl -s -O ${base_url}/${sha_file}
-                if sha256sum $sha_file ${CLOUD_IMAGE}.xz; then
-                    unxz ${CLOUD_IMAGE}.xz
-                    CLOUD_IMAGE=${CLOUD_IMAGE/.xz}
-                fi
+                curl -s -O ${base_url}/${CLOUD_IMAGE}.xz || die "failed to download ${base_url}/${CLOUD_IMAGE}.xz"
```

Comment:
here, and also in other sections, base_url and sha_file values are equally good without double quotes?

Reply:
Yes, but it can be added to be on the safe side.

Reply:
Added some more quotation marks as part of the script cleanup commit.